app/src/main/java/ch/hikemate/app/ui/profile/EditProfileScreen.kt

```diff
@@ -93,6 +98,8 @@ fun EditProfileScreen(
 
     var isLoading by remember { mutableStateOf(false) }
 
+    val coroutineScope = rememberCoroutineScope()
```

Comment:
It will create a coroutine scope that will only last as much as the Composable is used.
This would remove the coroutines and not do multiple go backs.

Reply:
Ooooh I see! Thanks!